tests/service_automation/pyproject.toml

```
@@ -43,5 +43,11 @@ markers = [
     "is-api: mark test as IS API test",
     "is-infra: mark test as IS infra test",
     "is-pipeline: mark test to be run in the IS pipeline",
-    "is-lambda: marks test as IS lambda related"
+    "is-lambda: marks test as IS lambda related",
+    "is-s3: marks test as IS S3 related",
+    "gp_search-api: marks test as gp search related",
```

Is it ok to stick to kebab-case here? (I love that term)

```suggestion
    "gp-search-api: marks test as gp search related",
```